querybook/server/models/query_execution.py

```diff
@@ -188,6 +188,29 @@ def to_dict(self):
         }
 
 
+class QueryExecutionMetadata(CRUDMixin, Base):
+    __tablename__ = "query_execution_metadata"
+
+    id = sql.Column(sql.Integer, primary_key=True)
+    query_execution_id = sql.Column(
+        sql.Integer, sql.ForeignKey("query_execution.id", ondelete="CASCADE")
+    )
+    sample_rate = sql.Column(sql.Float)
```

Comment:
will we add new columns for other metadata if we have in the future? or should we make it more general and just use a JSON column for all the metadata?

Reply:
I am okay with either, if we plan on add new columns for other metadata in the future we can keep separate table